LICENSE.txt

```diff
@@ -1,13 +1,60 @@
-Copyright 2018-2019 Element 84
+Copyright © 2020 United States Government as represented by the Administrator of the National Aeronautics and Space Administration. All Rights Reserved.
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+    http://www.apache.org/licenses/LICE NSE-2.0
```

Comment:
@DrewE84 ,

It looks like we have an errant space in the license URL.

Reply:
Good catch! I just pushed the fix.